File: test-suite/src/test/java/io/micronaut/guice/doc/examples/bindings/imported/CreditCardProcessorModule.java
package io.micronaut.guice.doc.examples.bindings.imported;

import com.google.inject.AbstractModule;

public final class CreditCardProcessorModule extends AbstractModule {
    @Override
    protected void configure() {
        bind(CreditCardProcessor.class).to(PayPalCreditCardProcessor.class);
    }
}

File: test-suite-kotlin/src/test/kotlin/io/micronaut/guice/doc/examples/bindings/imported/CreditCardProcessorModule.kt
package io.micronaut.guice.doc.examples.bindings.imported

import com.google.inject.AbstractModule

class CreditCardProcessorModule : AbstractModule() {
    override fun configure() {
        bind(CreditCardProcessor::class.java).to(PayPalCreditCardProcessor::class.java)
    }
}
